src/middleware.py

```diff
@@ -1,51 +1,22 @@
-from fastapi import Request, status
-from fastapi.responses import Response, JSONResponse
+import uuid
+from fastapi import Request
```

Comment:
A good practice is to divide three lavels of import with blank lines in the following order:
- standard library modules
- third-party packages
- internal project modules

Reply:
I did't know about it, only about order.